service/src/tests.rs

```diff
@@ -773,43 +773,53 @@ mod tests {
         assert_eq!(result.result.len(), 2);
     }
 
+    // test repeats initTopicAndSubscribeNode method from pubsub api
     #[marine_test(config_path = "../Config.toml", modules_dir = "../artifacts/")]
     fn init_topic_and_subscribe_node_test() {
         clear_env();
-        let key_str = "some_key".to_string();
+        let topic = "some_key".to_string();
         let timestamp = 123u64;
         let weight = 8u32;
         let pin = false;
         let value = "some_value".to_string();
         let peer_id = "some_peer_id".to_string();
         let mut cp = get_correct_timestamp_cp(1);
         cp.init_peer_id = peer_id.clone();
-        register_key_and_check!(aqua_dht, key_str, timestamp, pin, weight, cp);
+
+        // register topic
+        register_key_and_check!(aqua_dht, topic, timestamp, pin, weight, cp);
 
         let mut cp = get_correct_timestamp_cp(2);
         cp.init_peer_id = peer_id.clone();
 
-        let result = put_host_value_and_check!(aqua_dht, key_str, value, timestamp, vec![], vec![], 0u32, cp);
+        // make node a subscriber to it
+        let result = put_host_value_and_check!(aqua_dht, topic, value, timestamp, vec![], vec![], 0u32, cp);
         assert!(result.success);
 
         // clear db to imitate other node
         clear_env();
+
+        // notify neighbor about subscription
         let mut cp = get_correct_timestamp_cp(1);
         cp.init_peer_id = peer_id.clone();
```

Comment:
how does this code notify neighbour?

Reply:
https://github.com/fluencelabs/aqua-dht/blob/main/aqua/pubsub.aqua#L53
register topic and put value with `peer_id` = host_id